src/pages/news-announcements.tsx

```diff
@@ -137,8 +138,18 @@ export async function getServerSideProps() {
     data: { articles },
   } = await client.query({
     query: GET_INTERNAL_NEWS_CAROUSEL_ARTICLES,
+    variables: {
+      publishedDate: DateTime.now(),
+    },
```

Comment:
Here is where I am passing current time to the GraphQL query to limit articles found in the carousel.